src/index.html

```diff
@@ -8,6 +8,37 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div class="main">
+      <div class="card" data-qa="card">
+        <img
+          alt = "APPLE A1419 iMac 27&quot; Retina 5K (MNED2UA/A)"
+          class = "card__image"
+          src="images/imac.jpeg"
+        >
+        <h2 class="card__name">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</h2>
+        <p class="card__code">Product code: 195434</p>
+        <div class="card__ratings">
+          <ul class="card__stars-4">
+            <li class="card__star"></li>
+            <li class="card__star"></li>
+            <li class="card__star"></li>
+            <li class="card__star"></li>
+            <li class="card__star"></li>
+          </ul>
+          <p class="card__reviews">Reviews: 5</p>
+        </div>
+        <div class="card__price">
+          <div class="card__label">Price:</div>
+          <h1 class="card__value">$2,199</h1>
```

Comment:
```suggestion
          <p class="card__value">$2,199</p>
```
The h1 tag is a semantic tag for headings and should be single on the page. The price is not heading and the card element wouldn't be the single card.